region/maridia/inner-green/East Sand Hall.json

```diff
@@ -679,6 +679,28 @@
         "A little higher run speed can also work but may require releasing jump near the peak to avoid getting caught on the overhang."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "Cross-Room Blue Speed Jump",
+      "entranceCondition": {
+        "comeInGettingBlueSpeed": {
+          "length": 1,
+          "openEnd": 1,
+          "minExtraRunSpeed": "$4.4"
+        }
+      },
+      "requires": [
+        "canPlayInSand",
+        "canCrossRoomJumpIntoWater",
+        "canInsaneJump"
```

Comment:
This strat would be easier than the cross room speed jump, in expert

Reply:
Makes sense. So we can put the new notable in Expert: "Cross-Room Blue Speed Jump (Right to Left)". I did also test it again left-to-right but found it doesn't really work, or at least not any better than just doing the (Extreme) regular speedy jump over the Evirs. The trouble with left-to-right is its hard to get high enough to clear the pillar at the end.